OpenDataPolicingNC/Traffic-Stops
# ODPM-117: census data front-end

This patch revises the Census code in two ways:

* refactored according to the pattern of the other front-end base classes so as to be reusable across states
* new implementation of data-pulling and -processing logic to remove reliance on dummy census data for NC

It also adds an MD implementation of the Census base classes and adds them to the templates for MD.

To verify:

* Visit an MD agency that has census data (e.g. Aberdeen) and verify that census graphs appear and look reasonable and there are no JS console errors.
* Visit an NC agency previously known to have census data (e.g. Asheville) and verify that census graphs still appear and there are no JS console errors.

*Note:* NC data may need to be re-imported to get geo-data attached to agencies. Alternatively, a GEOID value for an agency can be looked up in [this table](https://github.com/OpenDataPolicingNC/Traffic-Stops/blob/dev/nc/data/NC_agencies.csv) and manually attached to an agency in the admin panel.
